rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```diff
@@ -106,6 +106,91 @@
         }
       }
     },
+    "type": {
+      "title": "Item type",
+      "type": "string",
+      "enum": [
+        "standard",
+        "issue"
+      ],
+      "default": "standard"
+    },
+    "issue": {
+      "title": "Issue information",
+      "type": "object",
+      "required": [
+        "display_text",
+        "received_date",
+        "expected_date",
+        "regular",
+        "status"
+      ],
+      "propertiesOrder": [
+        "display_text",
+        "received_date",
+        "expected_date",
+        "status"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "status": {
+          "type": "string",
+          "title": "Issue status",
```

Comment:
Title should come before type. See [guidelines, section order](https://github.com/rero/developer-resources/blob/master/guidelines/json-schemas.md#order).